src/issues/issueFeatureRegistrar.ts

```diff
@@ -488,11 +488,28 @@ ${labelLine}\n
 ${body ?? ''}\n
 <!-- Edit the body of your new issue then click the ✓ \"Create Issue\" button in the top right of the editor. The first line will be the issue title. Assignees and Labels follow after a blank line. Leave an empty line before beginning the body of the issue. -->`;
 		await vscode.workspace.fs.writeFile(bodyPath, this.stringToUint8Array(text));
-		const editor = await vscode.window.showTextDocument(bodyPath);
 		const assigneesDecoration = vscode.window.createTextEditorDecorationType({ after: { contentText: ' Comma-separated usernames, either @username or just username.', fontStyle: 'italic', color: new vscode.ThemeColor('issues.newIssueDecoration') } });
 		const labelsDecoration = vscode.window.createTextEditorDecorationType({ after: { contentText: ' Comma-separated labels.', fontStyle: 'italic', color: new vscode.ThemeColor('issues.newIssueDecoration') } });
-		editor.setDecorations(assigneesDecoration, [new vscode.Range(new vscode.Position(2, 0), new vscode.Position(2, assigneeLine.length))]);
-		editor.setDecorations(labelsDecoration, [new vscode.Range(new vscode.Position(3, 0), new vscode.Position(3, labelLine.length))]);
+		const editorChangeDisposable = vscode.window.onDidChangeActiveTextEditor((textEditor => {
+			if (textEditor?.document.uri.scheme === NEW_ISSUE_SCHEME) {
+				const assigneeRange = new vscode.Range(new vscode.Position(2, 0), new vscode.Position(2, assigneeLine.length));

```

Comment:
This seems to be inserting the decoration in front of the typed text:
![Screen Shot 2020-07-01 at 2 33 18 PM](https://user-images.githubusercontent.com/3672607/86293673-f30efe00-bba7-11ea-968c-08b63a9fc5d0.png)

I think we need to check the current line length instead of using `assigneeLine.length`

Reply:
Thank you for catching this!